ui/intermittents-view/ui/App.jsx

```diff
@@ -0,0 +1,33 @@
+import React from "react";
```

Comment:
@camd I know the test-view does this, but what are your thoughts on flattening the directory structure so `ui/intermittents-view/ui/App.jsx` becomes ` ui/intermittents-view/App.jsx`? (Having `ui/` in the path twice seems a little redundant)

Reply:
Yeah, I agree.  That sounds better.  :)